server/middleware/error-handler.js

```diff
@@ -0,0 +1,15 @@
+var expressErrorHandler = require('errorhandler');
+expressErrorHandler.title = 'Loopback';
+
+module.exports = errorHandler;
+
+function errorHandler(options) {
+  if (!options || options.includeStack !== false) {
+    return expressErrorHandler(options);
+  } else {
+    return function errorHandler(err, req, res, next) {
+      delete err.stack;
+      return expressErrorHandler(options)(err, req, res, next);
```

Comment:
Creating a new `expressErrorHandler(options)` for each request is suboptimal, it's better to call it only once. I'll fix this myself to speed things up.